samples/join-team-using-qr-code/nodejs/Readme.md

```diff
@@ -0,0 +1,105 @@
+---
+page_type: sample
+description: This sample demos joining a team using QR code containing the team id.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "21-12-2021 23:35:25"
+---
+
+# Join team using QR code sample
+
+This sample demos joining a team using QR code containing the team id.
+
+The user can Generate a new QR code (contains team id information) and then scan the QR code to join the team.
+
+`Currently, Microsoft Teams support for QR or barcode scanner capability is only available for mobile clients`
```

Comment:
`Currently, Microsoft Teams support for QR or barcode scanner capability is only supported for mobile clients`